Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/ScientificNotationErrors.dfy

```diff
@@ -0,0 +1,43 @@
+// RUN: %exits-with 2 %resolve "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+// Test error cases for scientific notation, trailing-dot shorthand, and leading-dot shorthand
+
+method MalformedScientificNotation() {
```

Comment:
Great to have these tests for malformed literals. Add tests where there is whitespace before a trailing dot or after a leading dot.